### DISCUSSION TITLE
Hit error meter is kinda... in the way

### DISCUSSION THREAD

**Discussion by Morilli** [ppy/osu#14882] [Ideas] [answered]
I'm noticing it mostly in mania, but it becomes apparent in some osu!standard maps as well (given this is an intentionally low placed slider, but still):
![osu! Screenshot 2021 09 28 - 22 25 47 41](https://user-images.githubusercontent.com/35152647/135162605-390c7956-396e-4a20-9d58-5eae3b262b9e.png)

The bar overlaps with normal hitobjects, or in the case of mania sitting above the bottom part.
![mania hit error meter](https://user-images.githubusercontent.com/35152647/135162934-68f5bedb-d50d-4270-ab14-ae7a75f6968e.png)

Wasn't there an option to disable or move this bar at some point? At least moving it lower when the progress bar graph is disabled would probably also help.

**Comment by peppy** [comment, marked as answer]
you can move it using the skin editor

**Reply by Morilli** [reply]
That... would actually work and suit my needs (except for having to do it for every skin I guess, but that's probably fine), if the skin editor didn't crash upon clicking save...

**Reply by smoogipoo** [reply]
Provide logs?

**Reply by peppy** [reply]
https://github.com/ppy/osu/discussions/14884 for reference
